### PR TITLE
Update 404 PageNotFound Style

### PR DESCRIPTION
**Pull request by mikebrandin**
This PR updates the 404 page to have a more consistent style with the rest of the website. The changes have been tested both locally and deployed on CloudFront. Builds off of #59.

You can see these changes seen [here](https://ddyr59f88ghkk.cloudfront.net/@#@##@).

As you mentioned, I have rebased and consolidated my git history and removed unnecessary changes for this PR.

### CODE REVIEW

File: site/visitor-console/src/pages/PageNotFound.tsx
Comment (ophilli) [inline site/visitor-console/src/pages/PageNotFound.tsx]: This is nitpicky, but can you `git mv site/visitor-console/src/pages/PageNotFound.tsx site/visitor-console/src/pages/NotFoundPage.tsx` so that the filepath matches the component name? Code is write-once, read-many, and little things like this will hopefully make it easier for future devs to connect the dots quicker.

**Review by ophilli — CHANGES_REQUESTED** [review]

File: site/visitor-console/src/pages/PageNotFound.tsx
Comment (mikebrandin) [reply site/visitor-console/src/pages/PageNotFound.tsx]: I have made those changes in my latest commit.

**Comment by ophilli** [comment]
Do you still want to merge this? If so you need to rebase & rebuild the site.

**Comment by mikebrandin** [comment]
Yes, I still would like to merge this PR because the current 404 page does not match the rest of the site. I have rebased the branch, renamed the files, and updated the build folder. Please let me know if there are any other issues with this PR preventing it from being merged.

**Review by dggorsen — APPROVED** [review]
Can you just rebase this for me before I merge? Feel free to ping me on discord when its been rebased

**Comment by mikebrandin** [comment]
This PR resolves issue #19

**Pull request merged**
